GoogleTestAdapter/TestAdapter/CTestPropertySettings.xsd

```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xs:schema elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="CTestProperties" nillable="true" type="CTestPropertySettingsContainer" />
```

Comment:
>CTestProperties [](start = 20, length = 15)

Would it be possible to add a test case for the new runsettings?

Reply:
I'll look into that